server/src/domain/media/media.service.ts

```diff
@@ -93,6 +93,18 @@ export class MediaService {
     return true;
   }
 
+  async handleGenerateThumbhashThumbnail({ id }: IEntityJob) {
```

Comment:
missing type annotation for boolean return

Reply:
Alright. The other functions don't have those type annotations.

Reply:
We use explicit return types some places, but not everywhere. Sometimes it is unnecessary IMO since the return type is inferred by the compiler when it's omitted. I don't have strong opinions if we include or exclude it.

Reply:
I feel like types should be set everywhere if possible, partly for legibility but also for safety. It returns a boolean right now, but it's not infeasible that a future revision may accidentally change this. Not sure the point that the compiler can automatically infer it is valid, as I'm sure TypeScript could automatically infer types in all sorts of places where it shouldn't. Is it possible to add a linting rule for explicit return types? Might help.

Reply:
If the return type changes from a boolean to something else, it will cause compilation problems for the functions that call it, if they use the return value. Right now the only place this method is used requires it to return a boolean. Regardless of if the return type is explicitly listed or not it will not compile if that was missing.

So, you get all the type safety features either way. On the other hand, if the return type was null | boolean, that would still be a valid return type even if null could never be returned. When the return type is listed explicitly sometimes you run into issues when it becomes outdated and becomes too general and not specific enough to what the method actually returns. They can lead to null checking in places where the value could never be null, etc.